StitchCore/Sources/StitchCore/Internal/Net/FoundationHTTPTransport.swift

```diff
@@ -56,7 +56,7 @@ public final class FoundationHTTPTransport: Transport {
 
         guard let response = finalResponse else {
             guard let err = error else {
-                throw StitchErrorCode.unknown
+                throw StitchError.serviceError(withMessage: nil, withServiceErrorCode: .unknown)
```

Comment:
[q] What's the case for this happening? Maybe we can have a better message?

Reply:
We should always be supplying some kind of message.